README.md

```diff
@@ -633,23 +633,35 @@ For additional information, see [here](https://www.zephyrproject.org/doc/getting
 There is only partial support for modules on Linux compared to Zephyr. Any hardware
 specific module (I2C, UART, GPIO, ADC etc.) is not supported on Linux. Trying
 to run a Zephyr specific module on Linux will result in the JavaScript not running
-successfully. Below is a complete table of modules and target support.
-
-| Module    | Linux                    | Zephyr                   |
-| :---:     | :---:                    | :---:                    |
-|  ADC      | <ul><li>- [ ] </li></ul> | <ul><li>- [x] </li></ul> |
-|  PWM      | <ul><li>- [ ] </li></ul> | <ul><li>- [x] </li></ul> |
-|  GPIO     | <ul><li>- [ ] </li></ul> | <ul><li>- [x] </li></ul> |
-|  I2C      | <ul><li>- [ ] </li></ul> | <ul><li>- [x] </li></ul> |
-|  BLE      | <ul><li>- [ ] </li></ul> | <ul><li>- [x] </li></ul> |
-|  UART     | <ul><li>- [ ] </li></ul> | <ul><li>- [x] </li></ul> |
-| Sensor    | <ul><li>- [ ] </li></ul> | <ul><li>- [x] </li></ul> |
-| Buffer    | <ul><li>- [x] </li></ul> | <ul><li>- [x] </li></ul> |
-| Console   | <ul><li>- [x] </li></ul> | <ul><li>- [x] </li></ul> |
-| Event     | <ul><li>- [x] </li></ul> | <ul><li>- [x] </li></ul> |
-| OCF       | <ul><li>- [x] </li></ul> | <ul><li>- [x] </li></ul> |
-|Performance| <ul><li>- [x] </li></ul> | <ul><li>- [x] </li></ul> |
-| Timers    | <ul><li>- [x] </li></ul> | <ul><li>- [x] </li></ul> |
+successfully. Below is a complete table of modules and target support. Some
+Zephyr targets listed are experimental and have not been fully tested. For this
+reason we have the following possibilities for support:
+
+* X - Supported
+* E - Experimental, not formally tested by QA but basic functionality verified.
+* NT - Not tested at all
+* Blank - Not Supported
+
+| Module    | Linux | Arduino 101 | K64F  | nRF52 | Arduino DUE | ST F411RE |
+| :---:     | :---: |    :---:    | :---: | :---: |    :---:    |   :---:   |
+|HelloWorld |   X   |      X      |   X   |    X  |      X      |     X     |
+|  ADC      |       |      X      |   X   |   NT  |      NT     |     NT    |
+|  PWM      |       |      X      |   X   |   NT  |      NT     |     NT    |
```

Comment:
Jimmy's right, I don't think we've ever tested PWM on K64F either. That should work but might require some pinmux changes.